products-service/interfaces/Product.ts

```diff
@@ -0,0 +1,8 @@
+export interface Product {
```

Comment:
btw, I noticed your naming is different (I mean some files are capitalized, while others - no).